scripts/base/frameworks/cluster/main.zeek

```diff
@@ -262,35 +298,52 @@ export {
 	global nodeid_topic: function(id: string): string;
 }
 
-global active_worker_ids: set[string] = set();
+# Track the names of fellow nodes each node should connect to
+global nodes_awaited: set[string] = set();
```

Comment:
That name is pretty confusing: It's the set of nodes we're expecting to see a `node_up` for, correct?

Some suggestions would be `nodes_expected`, `nodes_expected_up`, maybe `nodes_awaiting_up`.

"nodes awaited" to me means  the set of "nodes we have waited for" which is the opposite of what it's used for, but may come from a previous implementation.

> Track the names of fellow nodes each node should connect to

Track the names of peers we're expecting to connect/peer ?

Reply:
> That name is pretty confusing: It's the set of nodes we're expecting to see a node_up for, correct?

No, eventually we should see a `node_up` for nodes that we connect to as well as nodes that connect to us. `nodes_awaited` are nodes that we await, i.e. the local node connects to (see `setup-connections.zeek`).

Reply:
I should've left the part in my comment about not being a native speaker ;-) I still find it confusing, but this might be me.